test/cypress/support/faker.js

```diff
@@ -13,6 +13,16 @@ export const randomEmail = (directSignIn = true) => {
   return directSignIn ? `oc-test-${randomID}@opencollective.com` : `oc-noDirectSignIn-${randomID}@opencollective.com`;
 };
 
+export const randomGmailEmail = (directSignIn = true) => {
+  const randomID = uuidv4().split('-')[0];
+  return directSignIn ? `test-${randomID}@gmail.com` : `oc-noDirectSignIn-${randomID}@gmail.com`;
+};
+
+export const randomHotMail = (directSignIn = true) => {
+  const randomID = uuidv4().split('-')[0];
+  return directSignIn ? `test-${randomID}@outlook.com` : `oc-noDirectSignIn-${randomID}@outlook.com`;
```

Comment:
Only emails like `*@opencollective.com` can benefit from direct sign-in, we can safely remove this parameter

```suggestion
  return `oc-test-email-${randomID}@outlook.com`;
```